docs/guides/audio-config.md

```diff
@@ -54,6 +53,17 @@ An example to get PipeWire working on Ubuntu, which uses PulseAudio by default i
 The "Audio Volume" dialog / Audio in System Settings allow users to "Raise maximum volume", allowing to go past 100%. This
 does not offer a great deal of flexibility, it might work for getting acceptable recordings however.
 
+## Speakers
+
+All of apple's fancy tuning of the speakers is done in macOS, similar configuration is currently available only for MacBook Pro 16 inch 2019 only.
+
+### Macbook pro 16,1 2019
+
+Currently we have an experimental DSP(Digital Signal Processing) config for Macbook Pro 16,1 2019 with 6 speakers.
```

Comment:
maybe note that it does/doesn't work on 16,4 or that it's untested if the same config works on 16,4?

Reply:
I'd prefer MacBook Pro 16 inch, 2019 here

Reply:
I prefer to leave it to the exact model
MacBook Pro 16,1 2019

Reply:
About untested models. I'm already saying any other model is not supported and this configuration should not be used for now.

Reply:
But they are same models.

Reply:
what's the different betwen 16,1 and 16,4?

Reply:
The AMD graphics card

Reply:
https://support.apple.com/en-in/HT201300

Reply:
done